### PR TITLE
Update project to swift 2.0

### PR DESCRIPTION
**Pull request by artemSydorenko**

### CODE REVIEW

File: MenuExample/Controller/MenuViewController.swift [hunk 1]
```diff
@@ -19,6 +19,7 @@ class MenuViewController: UITableViewController {
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
 
+        self.view.backgroundColor = UIColor.clearColor()
```
Comment (maximletushov) [inline MenuExample/Controller/MenuViewController.swift]: @artemSydorenko this line of code is not related to the code upgrade to the swift 2.0. Do we really need this line ?

File: MenuExample/Controller/MenuViewController.swift [hunk 2]
```diff
@@ -33,18 +34,19 @@ extension MenuViewController {
...
+extension MenuViewController {
     override func tableView(tableView: UITableView, willSelectRowAtIndexPath indexPath: NSIndexPath) -> NSIndexPath? {
-        return indexPath == tableView.indexPathForSelectedRow() ? nil : indexPath
+        return indexPath == tableView.indexPathForSelectedRow ? nil : indexPath
     }
 
     override func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
         let rect = tableView.rectForRowAtIndexPath(indexPath)
-        var point = CGPointMake(CGRectGetMidX(rect), CGRectGetMidY(rect))
+        var point = CGPointMake(rect.midX, rect.midY)
```
Comment (maximletushov) [inline MenuExample/Controller/MenuViewController.swift]: Thanks

File: MenuExample/Controller/MenuViewController.swift (hunk 1 above)
Comment (artemSydorenko) [inline MenuExample/Controller/MenuViewController.swift]: iOS9 for some reason ignore background color for this VC when it set from storyboard. With white background animation looks ugly...
Maybe it because iOS9 still in beta

Comment (maximletushov) [inline MenuExample/Controller/MenuViewController.swift]: Thank you for explanation. Could you plz leave a TODO in code with explanation and info that we need to recheck when iOS9 will be in production. Thanks.

**Comment by maximletushov** [comment]
@artemSydorenko just add a TODO in code and good to merge. :shipit:
